src/main/kotlin/no/nav/syfo/sykmelding/kafka/producer/SendtSykmeldingKafkaProducer.kt

```diff
@@ -14,4 +14,14 @@ class SendtSykmeldingKafkaProducer(private val kafkaProducer: KafkaProducer<Stri
             throw e
         }
     }
+
+    fun tombstoneSykmelding(sykmeldingId: String) {
+        log.info("Tombstone sykmelding {}", sykmeldingId)
+        try {
+            kafkaProducer.send(ProducerRecord(topic, sykmeldingId, null)).get()
+        } catch (e: Exception) {
+            log.error("Kunne ikke skrive tombstone til bekreft-topic for sykmeldingid $sykmeldingId: {}", e.message)
```

Comment:
Brukes denne bare for bekreft-topic?

Reply:
Hmm, hva er forskjellen på denne og metoden i SykmeldingTombstoneProducer? :)

Reply:
Denne kom egentlig før tombstoneproducer og var for å tombstone bare sendte sykmeldinger

Reply:
Trenger vi begge..?